### PR TITLE
add hot-expresso recipe

### PR DESCRIPTION
**Pull request by cendekia**
Happy Hacktoberfest :tada: :beers:

### CODE REVIEW

**Comment by alfg** [comment]
Hi @cendekia, 

Happy Hacktoberfest!

Question: Should this be `hot-espresso` instead of `hot-expresso`?

File: src/recipes/hot-expresso.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+    "name": "Hot Expresso",
+    "description": "Morning boost expresso shots",
+    "github": "https://github.com/cendekia",
```
Comment (alfg) [inline src/recipes/hot-expresso.json]: Just your github username is fine:
```suggestion
    "github": "cendekia",
```

Comment (cendekia) [reply src/recipes/hot-expresso.json]: ahhh.. okay I will change it

**Comment by cendekia** [comment]
@alfg ok all set :)

**Comment by alfg** [comment]
Looks good! Thanks!

**Pull request merged**